docker/docker-compose.yml

```diff
@@ -3,6 +3,8 @@ version: '3.3'
 # Date: Feb 13 2021
 # Requirement: docker, docker-compose
 
+name: grottocenter
```

Comment:
@vmarseguerra I added this to display an appropriate name in Docker Desktop (when not specified, it's just "docker" and I don't know which "docker" is which when I'm working on various projects): 
![docker name](https://github.com/GrottoCenter/grottocenter-api/assets/20704943/afd8935b-3cfb-446d-b19a-3243cf9d6820)